certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/CertificateMetaData.java

```diff
@@ -20,4 +20,5 @@ public class CertificateMetaData {
   CareUnit careUnit;
   CareProvider careProvider;
   ResponsibleIssuer responsibleIssuer;
+  Staff creator;
```

Comment:
Consider updating this to createdBy, so that it better matches revokedBy.

Reply:
revokedBy isn't in this object. I was matching it against issuer. Should I leave it?